docs/connector-development/cdk-python/schemas.md

```diff
@@ -25,3 +25,64 @@ def get_json_schema(self):
     return schema
 ```
 
+## Schema normalization
+
+Complying output data  to declared json schema is important because those data later could be used by destination that rely on received data types (e.g. when data is stored in SQL database and you can't put INTEGER type into CHAR column and so on). In case of minor jsonschema mistake or API change could break synchronization process even if data fetching completed successfully.
```

Comment:
```suggestion
It is important to ensure output data conforms to the declared json schema. This is because the destination receiving this data to load into tables may strictly enforce schema (e.g. when data is stored in a SQL database, you can't put INTEGER type into CHAR column). In the case of changes to API output (which is almost guaranteed to happen over time) or a minor mistake in jsonschema definition, data syncs could thus break because of mismatched datatype schemas.
```